examples/ClarifaiPyspark_Example_NB.ipynb

```diff
@@ -0,0 +1,491 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "f048b9a5-0f6c-4a57-bf6e-d4c513c52d10",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Create a ClarifaiPyspark client object to connect to your App on Clarifai"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "3ff59442-b953-47e3-b50c-0eb11a8276f2",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "cspark = ClarifaiPySpark(user_id='user_id', app_id='app_id')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "eacc6ee9-7efb-48b6-9eb7-515ea8d01816",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Mention the dataset from your App on which you want to work.  \n",
+    "This creates a new dataset in the App if it doesn't already exist."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "cc9d473d-3f61-41d0-ab65-17c7dab8b8b6",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "dataset_obj = cspark_obj.dataset(dataset_id='dataset_id')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "6865d686-e89f-4b46-982e-138302bf8e69",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "#1. Upload Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "d8037987-2191-414a-b3ec-d4ac66e80094",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "### a. Upload from Volume folder   \n",
+    "If you have your dataset images/text files stored in databricks volume, you can directly upload the data files from volume to Clarifai App.  \n",
+    "The folder should contain only images.  \n",
+    "If the folder name is the label for all images inside it, then you can set labels paramter to True."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "39e4ae83-291e-4273-a422-750cda2d8039",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "dataset_obj.upload_dataset_from_folder(folder_path='volume_folder_path', input_type='image/text', labels=True/False)"
```

Comment:
This goes for through out notebook. Can we make this into two lines, github doesn't provide horizontal scroll render and viewing becomes difficult